### PR TITLE
head_ref_force_pushed event documentation

### LINKED ISSUE #11129: Missing documentation for `head_ref_force_pushed` issue event
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types

### What part(s) of the article would you like to see updated?

In the responses returned by the _Timeline events_ and _Issue events_ APIs for PRs that have been force-pushed, events of type `head_ref_force_pushed` can be found. However, this kind of event is not documented in the Issue event types page.

### Additional information
---
edited  by maintainer:
Here is [the writer's review](https://github.com/github/docs/issues/11129#issuecomment-964596043) for this issue.

### PR DESCRIPTION
**Pull request by IMPranshu**
### Why:

Closes https://github.com/github/docs/issues/11129

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@IMPranshu 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by IMPranshu** [comment]
My pleasure @ramyaparimi

**Comment by IMPranshu** [comment]
@ramyaparimi any changes needed in this?

**Comment by ramyaparimi** [comment]
@IMPranshu The PR is waiting for review from one of the writers on our team. A writer will be reviewing this PR soon ✨ 
Thanks so much for your patience 💖

File: content/developers/webhooks-and-events/events/issue-event-types.md
Comment (stevecat) [inline content/developers/webhooks-and-events/events/issue-event-types.md:325]:
```suggestion
The pull request's HEAD branch was force pushed.
```

**Review by guntrip — APPROVED** [review]
Thanks for making this change, @IMPranshu! I've made a change to the description and will merge this shortly ✨

**Comment by IMPranshu** [comment]
@stevecat Happy to contribute.

**Pull request merged**
